    Log message:
    
    PR tree-optimization/19865
    * tree-optimize.c (init_tree_optimization_passes): Run
    pass_may_alias after pass_sra.
    
--- gcc/gcc/tree-optimize.c	2005/02/14 21:32:14	2.71
+++ gcc/gcc/tree-optimize.c	2005/02/16 19:19:34	2.72
@@ -364,6 +364,10 @@
   NEXT_PASS (pass_ch);
   NEXT_PASS (pass_profile);
   NEXT_PASS (pass_sra);
+  /* FIXME: SRA may generate arbitrary gimple code, exposing new
+     aliased and call-clobbered variables.  As mentioned below,
+     pass_may_alias should be a TODO item.  */
+  NEXT_PASS (pass_may_alias);
   NEXT_PASS (pass_rename_ssa_copies);
   NEXT_PASS (pass_dominator);
   NEXT_PASS (pass_redundant_phi);
